### DISCUSSION TITLE
How to use ifconfig on ish?

### DISCUSSION THREAD

**Discussion by haazzzandsingning** [ish-app/ish#2027] [Q&A]
当我下载“net-tools”并想使用命令“ifconfig”时，正在发生一些事情："ifconfig：/proc/net/dev：没有这样的文件或目录  
ifconfig：ioctl 0x8912失败：不是tty"
and i want to know how to fix it......

**Comment by tower-town** [comment]
Hi! This is a duplicate of https://github.com/ish-app/ish/issues/145
